server/tests/e2e/user.e2e-spec.ts

```diff
@@ -1115,6 +1115,45 @@ describe('User', () => {
         });
     });
 
+	describe('PUT /api/user/maps/library/{mapID}', () => {
+		it('should add a new map to the local users library', async () => {
+			const mapLibraryEntry = await global.prisma.mapLibraryEntry.findFirst({
+				where: {
+					userID: user1.id,
+					mapID: map3.id }
+				}
+			);
+			expect(mapLibraryEntry).toBeNull();
+
+			await put({
+				url: `user/maps/library/${map3.id}`,
+				status: 200,
```

Comment:
This endpoint should be 204ing. Looks like missing a `@HttpCode(HttpStatus.NO_CONTENT)` below the `@Put` declaration, my bad.